rest-server/admin/src/main/scala/com/actionml/admin/MongoAdministrator.scala

```diff
@@ -72,15 +71,15 @@ class MongoAdministrator extends Administrator with JsonParser with Mongo {
     parseAndValidate[RequiredEngineParams](json).andThen { params =>
       engines = engines + (params.engineId -> newEngineInstance(params.engineFactory).initAndGet(json))
       if (engines(params.engineId) != null) {
-        if(enginesCollection.find(MongoDBObject("engineId" -> params.engineId)).size == 1) {
+        if (enginesCollection.find(MongoDBObject("engineId" -> params.engineId)).size == 1) {
           // re-initialize
-          logger.trace(s"Re-initializing engine for resource-id: ${params.engineId} with new params $json")
+          logger.trace(s"Re-initializing engine for resource-id: ${ params.engineId } with new params $json")
           val query = MongoDBObject("engineId" -> params.engineId)
           val update = MongoDBObject("$set" -> MongoDBObject("engineFactory" -> params.engineFactory, "params" -> json))
           enginesCollection.findAndModify(query, update)
         } else {
           //add new
-          logger.trace(s"Initializing new engine for resource-id: ${params.engineId} with params $json")
+          logger.trace(s"Initializing new engine for resource-id: ${ params.engineId } with params $json")
           val builder = MongoDBObject.newBuilder
```

Comment:
Let's not fix styling in PRs, it just makes them overly hard to read and we haven't agreed on style guidelines.